anomaly_detector/config.py

```diff
@@ -3,7 +3,7 @@
 
 import os
 import distutils
-
+from distutils import util
```

Comment:
Why did you add this import? Does not seem to be used (the import on line 5 is used)